src/DeviceRenderer.js

```diff
@@ -836,4 +844,18 @@ module.exports = class DeviceRenderer {
             this.emit(data.type, data.message);
         }
     }
+
+    destroy() {
+        document.removeEventListener('click', this.clickHandlerCloseOverlay.bind(this));
+        this.peerConnection?.removeEventListener('connectionstatechange', this.onConnectionStateChange);
+        this.mediaManager?.destroy();
+        this.gamepadManager?.destroy();
+        this.peerConnectionStats?.destroy();
+        delete this.peerConnectionStats;
+        delete this.video;
+        delete this.wrapper;
+        delete this.videoWrapper;
+        delete this.root;
+        delete this.stream;
```

Comment:
are we missing a delete this.peerConnection ?

Reply:
`this.peerConnection` is closed and set to null in the `disconnect` method, and I assumed that method was called since that's the case in Shadow. However I was wrong to assume this, it's simply not true here. I'll add a `disconnect` at the very begining of the `destroy` method.